lib/base-text-component.js

```diff
@@ -31,11 +31,11 @@ export default class BaseTextComponent extends Component {
 				self.setState({
 					value: maskedText
 				}, () => {
-					resolve(maskedText);
+					resolve({ maskedText, rawText: self.getRawValue() });
```

Comment:
Thanks for the contribution, but I need to change it a little bit.

It's computing the raw value on every change. It will cause performance issues.

I will add an option to check if it should include in the on change or not.